OrbitCore/LinuxTracingHandler.cpp

```diff
@@ -43,41 +42,23 @@ void LinuxTracingHandler::OnSchedulingSlice(SchedulingSlice scheduling_slice) {
   tracing_buffer_->RecordTimer(std::move(timer));
 }
 
-void LinuxTracingHandler::OnCallstack(
-    const LinuxTracing::Callstack& callstack) {
+void LinuxTracingHandler::OnCallstackSample(CallstackSample callstack_sample) {
```

Comment:
Ok, I think this answers my question about having one_of in CallstackSample proto.